### PR TITLE
Use env vars for setting up the DB; Update README

### PR DESCRIPTION
**Pull request by meganle**
I moved the database config from the `ormconfig.json` to env vars since this will be hard to change to deploy. Also, updated the README with more info on setting up the database and adding env vars

### CODE REVIEW

File: src/db/DbConnection.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// @flow
+import dotenv from 'dotenv';
+import { Connection, ConnectionOptions, createConnection } from 'typeorm';
+
+import Transaction from '../entities/Transaction';
+import User from '../entities/User';
+
+dotenv.config(); // establish env variables
```
Comment (young-k) [inline src/db/DbConnection.ts]: shouldn't this be moved to `server.ts`? also might be cleaner to have something like this in the long run: https://github.com/cuappdev/ithaca-transit-backend/blob/master/src/utils/EnvUtils.js

Comment (meganle) [reply src/db/DbConnection.ts]: We did this also for Pollo (separate file) also cause it can get pretty bloated once we add more models and migrations

Comment (young-k) [reply src/db/DbConnection.ts]: oh no i mean `dotenv.config`, we want to have that as early as possible

Comment (meganle) [reply src/db/DbConnection.ts]: oh yes, you are right

File: README.md [hunk 2]
```diff
@@ -23,11 +23,32 @@ After opening the project in your IDE, you can run it using
 npm start run
 ```
 
+### Setting up database:
+Make sure `PostgreSQL` is installed. After installation, start `PostgreSQL` and run the command
+````
```
Comment (kq-li) [inline README.md]: should this just be three?

File: README.md [hunk 3]
```diff
@@ -23,11 +23,32 @@ After opening the project in your IDE, you can run it using
...
+
 ### Environment Variables:
 It is recommended to use [`autoenv`](https://github.com/kennethreitz/autoenv). The required environment variables for this API are the following:
 ````bash
 export PORT=8000
-export ACCESS_TOKEN=FILL IN
+export ACCESS_TOKEN=FILL_IN
+export DB_USERNAME=FILL_IN
+export DB_PASSWORD=FILL_IN
+export DB_HOST=localhost
```
Comment (kq-li) [inline README.md]: & the appdev database droplet?

File: src/server.ts [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
 
 const app = new API();
 const server = app.getServer(false);
 const PORT: number = +process.env.PORT || 5000;
 const SERVER_ADDRESS: string = '0.0.0.0';
 
-createConnection().then(async (connection: any) => {
+dotenv.config(); // establish env variables
+
+DbConnection().then(async (connection: any) => {
```
Comment (kq-li) [inline src/server.ts]: i think we prefer DBConnection?

File: README.md (hunk 2 above)
Comment (meganle) [reply README.md]: changed them all to three lol

**Pull request merged**